src/web_app/js/call.js

```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-var Call = function(params) {
+var Call = function(params, doNotRequestMediaAndIceServers) {
```

Comment:
This is called in [appcontroller.js](https://github.com/webrtc/apprtc/blob/master/src/web_app/js/appcontroller.js#L148) but if it's not provided when creating the constructor, media and ice servers will be requested which was the default behavior before hence all places where it's called like before (just with params) the default behavior will be preserved.